site/data/charts/check-for-updates.json

```diff
@@ -108,6 +108,8 @@
     ["2024-06-01", 8, 28, 32, 651, 334, 1166, 5711, 382, 2162, 4677, 5654, 42935, 66254, 19047, 34613, 36566, 11873, 677091, 392085, 97995, 113097, ""],
     ["2024-07-01", 8, 56, 37, 596, 311, 955, 4061, 388, 2405, 5037, 5091, 29589, 61731, 16729, 51161, 39709, 10877, 783813, 440675, 180971, 142966, ""],
     ["2024-08-01", 4, 16, 19, 596, 269, 1282, 4087, 430, 2851, 4188, 4880, 31462, 70335, 15066, 38558, 44921, 10214, 945063, 448655, 207247, 182789, ""],
-    ["2024-09-01", 17, 26, 12, 481, 183, 1020, 3151, 293, 1419, 3584, 3355, 26725, 78210, 13581, 21400, 38657, 9518, 789838, 366952, 710356, 305501, ""]
+    ["2024-09-01", 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 78210, 13581, 21400, 38657, 9518, 789838, 366952, 710356, 305501, ""],
+    ["2024-09-01", 17, 26, 12, 481, 183, 1020, 3151, 293, 1419, 3584, 3355, 26725, 0, 0, 0, 0, 0, 0, 0, 0, 0, ""],
```

Comment:
Two Septembers?

Reply:
OK, I had to merge this manually before. I'll do that again, and try to fix the script 😁